docs/sources/operators-guide/mimir-runbooks/_index.md

```diff
@@ -1386,6 +1386,55 @@ How to **fix** it:
 
 - Increase the per-tenant limit by using the `-distributor.ha-tracker.max-clusters` option (or `ha_max_clusters` in the runtime configuration).
 
+### err-mimir-sample-timestamp-too-old
+
+This error occurs when the ingester rejects a sample because its timestamp is too old as compared to the most recent timestamp received for the same tenant across all its time series.
+
+How it **works**:
+
+- If the incoming timestamp is more than 1 hour older than the most recent timestamp ingested for the tenant, the sample will be rejected.
+
+How to **fix** it:
+
+- This error may occur when the system time of your Prometheus instance has been shifted backwards. If this was a mistake, fix the system time back to normal. Otherwise, wait until the system time catches up to the time it was changed, or delete all ingested samples with a timestamp later than system time.
+
+### err-mimir-sample-out-of-order
+
+This error occurs when the ingester rejects a sample because another sample with a more recent timestamp has already been ingested.
+
+How it **works**:
+
+- Currently, samples are not allowed to be ingested out of order as they must be stored in order and sorting is expensive. We are working on removing this limitation in the future.
+
+Common **causes**:
```

Comment:
Another common cause is what I described in this comment:
https://github.com/cortexproject/cortex/issues/2662#issuecomment-638654351

> Check for which metrics Cortex logs the "timestamp out of order" error and make sure the related Prometheus exporters are working correctly (ie. no clashing series after Prometheus relabelling

We could say that the error message contains information about the series whose sample is out of order. We recommend to ensure that there's no Prometheus relabelling configure, which cause series to clash after the relabelling.